### ISSUE TITLE
Implement abort event hotkey in test play mode

### ISSUE THREAD

**Issue by fmatthew5876** [EasyRPG/Player#2314]
If you run RPG_RT in test play mode, you can use the F10 key to abort the currently active foreground event at any time. 

We don't implement this feature at all. We currently use F10 for screenshots.

DynRPG address: 000B203A

**Comment by Ghabry** [comment]
Imo instead of using a hotkey for this it could be a debug menu option instead.

**Comment by fdelapena** [comment]
Because this is TestPlay only, F10 hotkey by default in this mode wouldn't hurt, and non-keyboard ports likely won't require the key mapping, but this is anyways possible once settings scene lands. Debug menu sounds fine as an option for all ports, also suitable for ports without F10 mapping but F9 mapped.

**Comment by Fishaphrodite** [comment]
Maybe I shouldn't poke my nose in this, however, as far as I understand, using the same shortcut as the debug menu would prevent accessing the debug menu during the event itself (while it doesn't sound problematic, I don't know how that would react during battles initiated by an event, also the test-play debug menu is a feature I like to be able to trigger and undo without cancelling the current event).

F10 is used for screenshots, so it'd be a bit off to change it in that regard as well (I spam the screenshot key a lot, if I were to break a chain of event by doing this, I would be a bit disturbed).

Debug menu option would be fine, but the debug menu cannot be accessed during a message box, so if for some reason the message box is stuck, that might be troublesome (However, that is assuming this issue could happen, so maybe that is a non-issue.)

I am just an humble user of the player however, so you may disregard my comments if they are too intrusive/wrong.

**Comment by fdelapena** [comment]
I agree the take screenshot key should be moved to another function key to prevent clashing with the F10 key to match RPG_RT behavior, despite this function is fairly unknown for most game developers.

**Comment by fmatthew5876** [comment]
I would be in favor of moving the screenshot key and replicating this behavior for F10.

**Comment by rueter37** [comment]
I would suggest to move the screenshot key to F7, as @fmatthew5876 already reserved the F8 key for the planned game settings scene (see #2035).

**Comment by fdelapena** [comment]
As mentioned on IRC, RPG Maker 2003 1.10+ uses  the F1 key, so it could be reused for settings and available in Player.

**Comment by carstene1ns** [comment]
<kbd>F7</kbd> for screenshots is fine for me. Also using the <kbd>print-screen</kbd> key would not be a good idea (could open another can of worms with thirdparty screenshot programs, e.g. `greenshot` under windows for example. Also some keyboards might not have it)
However we already have some inconsistencies anyway (→ steam <kbd>F12</kbd>).

**Comment by Ghabry** [comment]
Print screen also steals the focus on KDE (opens spectacle).

Just shifting it somewhere else is okay. 

Also F1 for settings (and credits) also sounds good
